contracts/exchange-wrappers/Starkware.sol

```diff
@@ -35,6 +61,56 @@ contract Starkware is I_StarkwareContract {
         uint256 quantizedAmount
     ) external override
     {
+        uint256 amount = quantizedAmount;
```

Comment:
Are you sure quantized amount is in wei? Based on the naming, it seems to have a special meaning... Maybe Ken knows more?

Reply:
see other comments